src/Tests/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/RazorSourceGeneratorTests.cs

```diff
@@ -1016,21 +1016,21 @@ private static GeneratorRunResult RunGenerator(Compilation compilation, ref Gene
 
         private class AppLocalResolver : ICompilationAssemblyResolver
         {
-            public bool TryResolveAssemblyPaths(CompilationLibrary library, List<string> assemblies)
+            public bool TryResolveAssemblyPaths(CompilationLibrary library, List<string>? assemblies)
             {
                 foreach (var assembly in library.Assemblies)
                 {
                     var dll = Path.Combine(Directory.GetCurrentDirectory(), "refs", Path.GetFileName(assembly));
                     if (File.Exists(dll))
                     {
-                        assemblies.Add(dll);
+                        assemblies!.Add(dll);
```

Comment:
instead add a check before the loop?
```
if (assemblies == null) return false
```

Reply:
on second thought if someone does pass null here it will blow up with NRE as before. disregard.